Source/Blazorise/Utilities/IO/IFileEntry.cs

```diff
@@ -78,5 +110,10 @@ public interface IFileEntry
         /// <param name="cancellationToken">A cancellation token to signal the cancellation of streaming file data.</param>
         /// <exception cref="T:System.IO.IOException">Thrown if the file's length exceeds the <paramref name="maxAllowedSize" /> value.</exception>
         Stream OpenReadStream( long maxAllowedSize = 512000, CancellationToken cancellationToken = default );
+
+        /// <summary>
+        /// Cancels any ongoing FileEntry operation.
+        /// </summary>
+        public void Cancel();
```

Comment:
Should the be awaitable with `Task`?

Reply:
I'll have to take a look. Maybe it happens in a place where it does not make snse to be a Task?

Reply:
Guess I knew the implementation would be simple and didn't incur on the async cost.
![image](https://user-images.githubusercontent.com/22283161/191357290-88ba6315-61de-49a3-aa5d-ce500a71802d.png)

I don't think its needed. Think we should make it return `Task` anyway?

Reply:
If it's a new API, then better to make async now.

Reply:
Done